### DISCUSSION TITLE
context search does not support people filter

### DISCUSSION THREAD

**Discussion by juxeii** [immich-app/immich#7432] [Q&A] [answered]
Hi,

I tried a search with one selected person **and** an additional context search.
![image](https://github.com/immich-app/immich/assets/4497675/f8e409a2-edc2-4cf0-a748-5c7a3f9478d7)
****
Is this a bug or a constraint?
IMHO selecting a face **and** providing context terms like `beach `is kind of a natural thing, isn't it?

**Comment by bo0tzz** [comment, marked as answer]
See #7388

**Comment by mertalev** [comment]
It's a technical constraint that's tricky to solve. See [this](https://github.com/immich-app/immich/discussions/7388#discussioncomment-8579842) for more context.
